[IMP] inventory: update DHL shipping connector config (#17509)
odoo/documentation: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dhl_credentials.rst

Reviewer: ```suggestion
   The *SiteID* and *Password* are different credentials from the ones used to log in to a DHL
```
correction

Reviewer: ```suggestion
If a DHL Express account is available, log in to the `DHL Developer portal
```
correction

Reviewer: ```suggestion
- :guilabel:`DHL Product`: The shipping service purchased from DHL (e.g., Express Worldwide).
- :guilabel:`DHL Package Type`: The type of DHL package used for delivery (e.g., DHL Box).
```
correction

Reviewer: ```suggestion
- :guilabel:`Package Weight Unit`: The unit of measure used to display package weight.
- :guilabel:`Package Dimension Unit`: The unit of measure used to display package size.
- :guilabel:`Label Format`: The file format used to generate shipping labels.
- :guilabel:`Label Template`: The paper size used to print shipping labels.
```
correction

Reviewer: ```suggestion
- :guilabel:`Region`: The region in which the shipping method is used.
```
correction

DHL integration
===============

Set up the DHL shipping connector in Odoo to manage DHL shipments to clients directly within Odoo.
Configuring a DHL shipping method simplifies the process of :doc:`calculating shipping rates
<../setup_configuration>` and :doc:`generating shipping labels <labels>`.

.. seealso::

>    :doc:`third_party_shipper`.

Reviewer 2: ```suggestion
   :doc:`third_party_shipper`
```

Configure the DHL shipping method
=================================

To enable DHL delivery methods, :ref:`install <general/install>` the **DHL Express Shipping**

> module. DHL delivery methods can be configured and added as a line item to sales orders (SOs), which

Reviewer 2: ```suggestion
module. DHL delivery methods can be configured and added as a line item to sales orders, which
```
nitpick: wondering if it's necessary to define the acronym if it isn't used in the rest of the doc?

> Configure DHL shipping method

Reviewer: ```suggestion
Configure the DHL shipping method
```
suggestion

Begin configuring the shipping method by entering its title in the :guilabel:`Shipping Method`
field.

In the :guilabel:`Provider` drop-down menu, select the :guilabel:`DHL` option, and a new
:guilabel:`DHL Configuration` tab appears at the bottom of the form.

All other fields in this section are identical on the shipping method forms for each shipping
carrier. See :doc:`third-party shippers <third_party_shipper>` for standard configuration
instructions.

DHL Configuration tab
---------------------

The :guilabel:`DHL Configuration` tab on the shipping method form is used to connect the user's DHL

> account to Odoo, and configure the shipping method's details.

Reviewer: ```suggestion
account to Odoo and configure the shipping method's details.
```
correction

Once the DHL connection is set up, use the smart buttons at the top of the form to publish, turn on

> production mode or activate debug logging.

Reviewer: ```suggestion
production mode, or activate debug logging.
```
suggestion

> - :guilabel:`Unpublished`/:guilabel:`Published`: determines if this shipping method is available on

Reviewer: ```suggestion
- :guilabel:`Unpublished`/:guilabel:`Published`: Determines if this shipping method is available on
```
correction

  the user's **eCommerce** website.

> - :guilabel:`Test Environment`/:guilabel:`Production Environment`: determines whether label creation

Reviewer: ```suggestion
- :guilabel:`Test Environment`/:guilabel:`Production Environment`: Determines whether label creation
```

>   is for testing and cancelled immediately (Test) or generate real shipping label that is charged to

Reviewer: ```suggestion
  is for testing and canceled immediately (Test) or generates a real shipping label that is charged to the
```
suggestion as I don't know if this is a GUI call out

  DHL account (Production).

> - :guilabel:`No Debug`/:guilabel:`Debug Requests`: determines whether API requests and responses are

Reviewer: ```suggestion
- :guilabel:`No Debug`/:guilabel:`Debug Requests`: Determines whether API requests and responses are
```
